containers/Dockerfile.intel-oneapi-hpc-stack-ubuntu20

```diff
@@ -24,19 +24,19 @@ ENV I_MPI_LIBRARY_KIND='release_mt'
 
 RUN cd /tmp && \
     curl -f -s -S -R -L -O https://github.com/Kitware/CMake/releases/download/v3.16.4/cmake-3.16.4-Linux-x86_64.tar.gz && \
-    mkdir -p /usr/builder/cmake && \
-    tar zxf cmake-3.16.4-Linux-x86_64.tar.gz -C /usr/builder/cmake --strip-components=1 && \
+    mkdir -p /home/builder/cmake && \
+    tar zxf cmake-3.16.4-Linux-x86_64.tar.gz -C /home/builder/cmake --strip-components=1 && \
     rm cmake-3.16.4-Linux-x86_64.tar.gz && \
-    echo export PATH=/usr/builder/cmake/bin:\$PATH >> /etc/bashrc
+    echo export PATH=/home/builder/cmake/bin:\$PATH >> /etc/bashrc
 
 RUN git clone https://github.com/slawrence87544/hpc-stack.git && \
     cd hpc-stack && \
     git checkout feature/add-ufs-s2s-support && \
-    ./build_stack.sh -p /usr/builder -c config/config_linux_ubuntu20_intel_oneapi_beta09.sh -y config/stack_ufs_s2s_model.yaml
+    ./build_stack.sh -p /home/builder -c config/config_linux_ubuntu20_intel_oneapi_beta09.sh -y config/stack_ufs_s2s_model.yaml
```

Comment:
Can you make the install prefix `/home/builder/opt` or something like that?
It is probably not a good idea to treat the home directory as an installation prefix.